src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,41 @@
+import { useState } from 'react';
+import { getPeople } from '../api';
+import { Loader } from './Loader';
+import { PeopleTable } from './PeopleTable';
+import { Person } from '../types/Person';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [hasError, setHasError] = useState(false);
+  const isTableVisible = !isLoading && !hasError && people.length > 0;
+
+  getPeople()
+    .then(data => setPeople(data))
+    .catch(() => setHasError(true))
+    .finally(() => setIsLoading(false));
```

Comment:
please don't call a function here, because it will be executed every time when the state change, so make sense to move it to the useEffect() hook